client/app/components/sections/Topbar/Topbar.css

```diff
@@ -54,16 +54,29 @@
     }
   }
 
+  & button {
```

Comment:
Those generic selectors within the Topbar.css file can be thought of being our CSS normalisation from the old styles.

That specific rule undoes the rounded corners that we have in all buttons in old styles, so I think it makes sense to keep quite generic and get the same effect to all future buttons in the Topbar.

Does this make sense to you?